src/phoneNumberInput/utils/pnoneNumberUtils.js

```diff
@@ -0,0 +1,94 @@
+import countries from '../data/countries';
+
+/**
+ *
+ * @param code
+ * @returns {*}
+ */
+export const findCountryByCode = code => {
+  if (code && code.length === 2) {
+    // Find longest matching prefix
+    const matchingCodes = countries.filter(country => code === country.iso2);
+
+    if (matchingCodes && matchingCodes.length) {
+      return matchingCodes.reduce((a, b) => (a.phone.length > b.phone.length ? a : b));
+    }
+  }
+  return undefined;
```

Comment:
```suggestion
  return null;
```

`null` indicates that it's set by us better than _defining_ something as `undefined`.